problems/arraysWithLoops.js

```diff
@@ -66,24 +111,38 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+const findSecondSmallest = (nums) => {
+  let secondSmallest = Math.min.apply(null, nums);
+  nums.indexOf(secondSmallest), 1
+  return secondSmallest +1;
```

Comment:
This won't really work. This only works if the secondSmallest happens to be plus one of the smallest.  Not [3, 5, 7] or most arrays. 
Please revisit the problem using two variables that can updated. 
Same for the problem below.